support/systems/README.md

```diff
@@ -0,0 +1,7 @@
+# Systems
+
+We are currently using [OTRS](https://otrs.rocket.chat/otrs/index.pl?Action=AgentDashboard) as our main support system.
+
+Customers can either create tickets directly on the platform or by sending an email to [support@rocket.chat](mailto:support@rocket.chat).
```

Comment:
I would add that the URL for customers to create a ticket is https://support.rocket.chat